ibis_tpc/h19.py

```diff
@@ -20,32 +20,32 @@ def tpc_h19(
 
     q1 = (
         (q.p_brand == BRAND1)
-        & (q.p_container.isin(("sm case", "sm box", "sm pack", "sm pkg")))
```

Comment:
How were these working before?  (or weren't they?)

Reply:
Apparently `sqlite` doesn't care about case at all?

Reply:
hmm, no wait, that's not right.  I have no idea how it was working